internal/hooks/spec.go

```diff
@@ -25,6 +25,13 @@ const (
 	// CreateRuntime is called immediately after the deprecated Prestart hook.
 	// CreateRuntime commands are called in the Runtime Namespace.
 	CreateRuntime HookName = "createRuntime"
+
+	// StartContainer hooks MUST be called before the user-specified process is executed as
+	// part of the start operation. This hook can be used to execute some operations in the
+	// container, for example running the ldconfig binary on linux before the container process
+	// is spawned. The StartContainer hooks' path MUST resolve in the container namespace.
+	// The StartContainer hooks MUST be executed in the container namespace.
+	StartContainer HookName = "startContainer"
```

Comment:
were you planning to use this, I don't think I've seen it.

Reply:
Hmm, I think I was planning to use it to run 'ldconfig' in the container on start but didn't do it here. I can take it out and add that part in a later PR.